packages/flutter/lib/src/rendering/proxy_box.dart

```diff
@@ -1546,7 +1546,7 @@ class RenderFittedBox extends RenderProxyBox {
     if (_transform != null)
       return;
 
-    if (child == null) {
+    if (child == null || size.isEmpty) {
       _hasVisualOverflow = false;
```

Comment:
In particular I suspect this will break localToGlobal for things inside a zero-size RenderFittedBox.